src/Entities/Enkelvoudiginformatieobject.php

```diff
@@ -66,6 +69,51 @@ public function size(): int
         return $this->getValue('bestandsomvang', 0);
     }
 
+    public function creationDate(): string
+    {
+        $date = $this->getValue('creatiedatum', '');
+
+        if (empty($date)) {
+            return '';
+        }
+
+        try {
+            $date = new DateTime($date);
```

Comment:
Als je toch al 'n try/catch block hebt, zou je de format() call hier ook in kunnen zetten. Bijv.:

```php
try {
    return (new DateTime($date))->format('d-m-Y');
} catch (Exception $e) { // $e zou je weg kunnen halen wanneer we alleen PHP >= 8 ondersteunen
    return '';
}
```